code/modules/templates/verbs.dm

```diff
@@ -0,0 +1,38 @@
+/client/proc/TemplatePanel()
+	set name = "Template Panel"
+	set category = "Debug"
+
+	// Place
+
+	if(!check_rights(R_FUN))
+		return 0
+
+	var/dat = "<center><span class='statusDisplay'>"
+
+	if(check_rights(R_FUN))
+		dat += "<a href='?_src_=holder;template_panel=1;action=place'>Place</a>"
+		dat += " | <a href='?_src_=holder;template_panel=1;action=upload'>Upload and Place</a>"
+
+	dat += "</span><br><br>"
+
+	if(length(SStemplate.placed_templates))
+		dat += "<table>"
+		dat += "<tr><th>Name</th><th>Position</th>[check_rights(R_FUN) ? "<th>Actions</th>" : ""]"
+
+		for(var/datum/dmm_object_collection/template in SStemplate.placed_templates)
+			dat += "<tr><td>[template.name]</td><td>{[template.location.x], [template.location.y], [template.location.z]}</td>"
+			if(check_rights(R_FUN))
```

Comment:
Now, this needs Headmin inputs more specifically, i am not sure what to assign them to, so i defaulted them to +FUN